### ISSUE TITLE
Icons Just Not Showing Up

### ISSUE THREAD

**Issue by olegbc1** [domtronn/all-the-icons.el#107]
For some reason I cannot make any of the icons display at all. Here is what they look like in Dired sidebar:
![icons in dired sidebar](https://user-images.githubusercontent.com/9222334/36122593-c93b60f2-0ffe-11e8-9b5a-a18296240098.png)

and it's the same if I put `all-the-icons-fileicon` or `all-the-icons-wicon` in scratch buffer. 

PS I've installed all-the-icons from Melpa-stable and am using emacs 25.3.1 on Windows 10. I downloaded the necessary [fonts ](https://github.com/domtronn/all-the-icons.el/tree/master/fonts) and installed them manually (by double clicking on each file and pressing "install")

Appreciate any pointers.

**Comment by jtbm37** [comment]
Have a look if this is your issue: https://github.com/jtbm37/all-the-icons-dired/issues/12

**Comment by olegbc1** [comment]
Thanks for the suggestion @jtbm37. However I had just updated to GNU Emacs 26.0.90 (build 3, x86_64-w64-mingw32) of 2017-10-1 and the issues persists.

**Comment by wjbg** [comment]
HI all, 

I have exactly the same problem and am also on Emacs 25.3.1 on Windows 10. This is a typical example of the dired buffer:

![dir](https://user-images.githubusercontent.com/29379051/36560925-7fc4b8c6-1812-11e8-9569-9f952451e3d6.PNG)

The odd thing is that this seems to be a dired problem, as the icons do show up nicely in the minibuffer when switching buffers: 

![ivy-minibuffer](https://user-images.githubusercontent.com/29379051/36560939-8a017496-1812-11e8-9650-3a71087c43e7.PNG)

Any ideas to make progress here would be greatly appreciated.

**Comment by jtbm37** [comment]
@wjbg what is your `M-x version` ?

**Comment by jtbm37** [comment]
```
(require 'dash)

(let ((fonts '("github-octicons" "all-the-icons" "fontawesome" "octicons" "weathericons" "file-icons")))
  (-filter (lambda (x) (member x fonts)) (font-family-list)))
```
Does this snippet also return anything ?

For me it returns this
```
("github-octicons"
 "all-the-icons"
 "file-icons")
```

**Comment by wjbg** [comment]
@jtbm37 

Thanks for your swift response!

Regarding your questions:

`M-x version` yields: `GNU Emacs 25.3.1 (x86_64-w64-mingw32) of 2017-09-26`

`(insert (all-the-icons-octicon "file-directory" :v-adjust all-the-icons-dired-v-adjust :face 'all-the-icons-dired-dir-face))` gives:

`#("" 0 1 (font-lock-ignore t rear-nonsticky t display (raise 0.012) face (:family "github-octicons" :height 1.2 :inherit all-the-icons-dired-dir-face)))`

```(require 'dash)

(let ((fonts '("github-octicons" "all-the-icons" "fontawesome" "octicons" "weathericons" "file-icons")))
  (-filter (lambda (x) (member x fonts)) (font-family-list)))`
```
returns nothing.

I hope this helps pinpoint the problem.

**Comment by jtbm37** [comment]
@wjbg If the last snippet returns nothing this means you do not have the `all-the-icons` fonts installed on your pc. Did you install these [fonts](https://github.com/domtronn/all-the-icons.el#resource-fonts)?

**Comment by wjbg** [comment]
@jtbm37 Sorry, my bad. I apparently did not evaluate the last expression properly. After confirming that the fonts were actually installed, I tried the snippet again (and this time properly). It yields:

`("github-octicons" "file-icons" "all-the-icons" "file-icons" "all-the-icons")`

So that seems to be alright.

**Comment by fpopineau** [comment]
Hi there,
I'm also using emacs 26.0.91 repository version "9ff763484212d25cbdfc2f5921ba1e7bbdbdab10"
under Windows 10 and I have the same problem. No icons are displayed anymore.
It used to work some time ago with a previous version of emacs.

I can confirm that if I evaluate (all-the-icons-insert-alltheicon) in the *scratch* buffer,
I get a list with the icons in the minibuffer.
How once inserted into the *scratch* buffer, I don't get the icon anymore.

Looks like something has changed in emacs recently and maybe this is even a bug (?).

**Comment by fpopineau** [comment]
The problem is fixed with

(load-library "font-lock+")

Best regards,

2018-03-07 13:44 GMT+01:00 Wojciech Polak <notifications@github.com>:

**Comment by frondeus** [comment]
Thank you! It works fine now:

I downloaded font-lock+ from emacswiki and inserted in my config:

```elisp
(require 'font-lock)
(require 'font-lock+)
(use-package all-the-icons)
(use-package all-the-icons-dired
  :hook (dired-mode . all-the-icons-dired-mode))
```

**Comment by jtbm37** [comment]
@fpopineau Nice one! Thank you for sharing.

**Comment by wjbg** [comment]
Great stuff! Thanks @fpopineau !

**Comment by wyuenho** [comment]
Fixed by #106
